index.html

```diff
@@ -281,7 +282,39 @@ <h2>Skills</h2>
         </div>
 
     </section>
+    <div class="div"><br><br><br></div>
+    <!-- Contact Us Section -->
+    <div class="div">
+        <section class="contact">
+            <div class="contact-items">
+                <h2>Contact me</h2>
+            </div>
+            <div class="contact-items">
+                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it!</p>
 
+            </div>
+            <div class="contact-items">
+                <form action="https://formspree.io/f/xeqwnkae" method="post" class="form-flex">
+                    <div class="form-flex-item ">
+                        <input type="text" placeholder="Lorem Ipsum" maxlength="30" name="userName">
+                    </div>
+                    <div class="form-flex-item ">
+                        <input type="emial" placeholder="loremipsum@gmail.com" name="emial">
+                    </div>
+                    <div class="form-flex-item textarea">
+                        <textarea name="" id="" cols="30" rows="12" maxlength="500" required name="text">Write your message here...
```

Comment:
- [ ] [OPTIONAL] Great work including your `<textarea>` field here 💯. However, please kindly consider deleting the text "write your message here..." from the tag and include that text as a placeholder property instead. 🤓 This change will enhance the user experience of your form.